app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -300,33 +300,33 @@ public void cropCapturedImage(Uri picUri){
         cropIntent.putExtra("return-data", true);
         startActivityForResult(cropIntent, CROP_PICTURE);
     }
-    public void setChatBackground(){
+
+    public void setChatBackground() {
         shre = PreferenceManager.getDefaultSharedPreferences(this);
-        String previouslyChatImage = shre.getString("image_data", "");
+        String previouslyChatImage = shre.getString(Constant.IMAGE_DATA, "");
```

Comment:
@the-dagger, So that I have made Constant.java a helper to manage Keys values.
It's not in strings.xml